tls/s2n_send.c

```diff
@@ -167,10 +168,14 @@ ssize_t s2n_sendv_with_offset(struct s2n_connection *conn, const struct iovec *b
                 cbcHackUsed = 1;
             }
         }
+    
+        GUARD(s2n_stuffer_rewrite(&conn->out));
 
+        /* Check if encryption key limit has been reached */
```

Comment:
Nit: remove or rephrase this comment. Theoretically s2n_post_handshake_send does other things to, based on its name.